client/my-sites/controller.js

```diff
@@ -267,7 +267,7 @@ function createSitesComponent( context ) {
 	const contextPath = sectionify( context.path );
 
 	// This path sets the URL to be visited once a site is selected
-	const basePath = contextPath === '/sites' ? '/stats' : contextPath;
+	const basePath = contextPath === '/sites' ? '/home' : contextPath;
```

Comment:
Shouldn't this rely on the same logic as elsewhere?

Reply:
I notice that it redirects, so it will still work as expected. My concern is more that in the future we might decide to change the landing page for /sites again and it would be really nice if we could change it in one place rather than lots as you've had to do here.

I'll add a commit to demonstrate my idea.

Reply:
@scruffian I had the same intuition! Unfortunately the selector logic that depends on site information doesn't work here. Because we're seeing a list of sites, there's no selected site we can query to determine if Customer Home is available.

To determine the url based on site data, I think we'd need to refactor the component to generate the url after the site is selected. I'm going to save that for another day!

Reply:
Ok good call :)